pss/pss.go

```diff
@@ -52,11 +59,41 @@ func (p *Pss) Send(ctx context.Context, targets [][]byte, topic trojan.Topic, pa
 		return nil, err
 	}
 
+	tag := chunk.NewTag(1, "pss-chunks-tag", 0, false)
+	tc.WithTagID(tag.Uid)
+
 	// SAVE trojanChunk to localstore, if it exists do nothing as it's already peristed
-	// TODO: for second phase, use tags --> listen for response of recipient, recipient offline
 	if _, err = p.localStore.Put(ctx, chunk.ModePutUpload, tc); err != nil {
 		return nil, err
 	}
+	tag.Inc(chunk.StateStored)
+	tag.Total = 1
+
+	monitor := &Monitor{
+		chunk:  tc,
+		tag:    tag,
+		states: make(map[chunk.State]string),
+	}
+
+	monitor.updateState()
 
-	return tc, nil
+	return monitor, nil
+}
+
+func (m *Monitor) updateState() {
```

Comment:
[my previous comment] about these strings applies here as well, could you please explain what purpose these serve?

Reply:
Same reply as [before](https://github.com/ethersphere/swarm/pull/2159#discussion_r409674481)